### ISSUE TITLE
This plugin will highlight `I爱U` as a word, but vim's motion regard it as 3 words.

### ISSUE THREAD

**Issue by sisrfeng** [dominikduda/vim_current_word#30]
>   Helps predict how vim word movements (`w`,  `e`,  `b`) will behave
> Plays well with * and #

However, this plugin will highlight `I爱U` as a word,  ( because `@` is in iskeyword? `I-U` will not be highlighted as a word ,because `-` is not in my `iskeyword` (iskeyword=@,48-57,_) .  )  But nvim' s word motions, e.g., `w` , `*`, and `#`  regard `I爱U` as 3 words.

**Comment by Freed-Wu** [comment]
> because @ is in iskeyword

Yes, `matchadd('CurrentWord', '\k*\%#\k*', ...)` respect `iskeyword`, so if `@` is in `keyword`, it is a normal behaviour.

**Comment by sisrfeng** [comment]
Is it possible to make the behavior of *highlight* match the behavior of *motion* ? E.g., highlight `I爱U` as a word and have a key which can move to the next or previouw `I爱U` ?

**Comment by Freed-Wu** [comment]
> have a key which can move to the next or previous

Currently, this plugin doesn't have any keys.
